### PR TITLE
Introduce email header auto response page for triggers.

### PR DESCRIPTION
**Pull request by dominikklein**
Added a small section about the new possibility to set the email header for `x-zammad-is-auto-response` and `x-zammad-send-auto-response` directly in the email.

### CODE REVIEW

**Comment by MrGeneration** [comment]
This is to enforce hard states for e.g. auto responses to be noticed as true or false, right?
So basically a way to overwrite the default detection in case you still want an auto response as example?

**Comment by MrGeneration** [comment]
Hey @martini - please have a look on the technical parts of the header section. 
I did verify them but would love you to double tab. 🙏

From my side this one's ready as soon as Jessica is happy with the wording. :)

**Review by MrGeneration — APPROVED** [review]

File: channels/email/email-headers.rst [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+         This header is not available for follow ups.
+
+Article attributes
+------------------
+
+If needed Zammad allows you to manipulate attributes or states of fetched
+email articles.
+
+``X-Zammad-Article-Sender``
+   | Manipulate the sender type (Agent, Customer or System)
```
Comment (techerella) [inline channels/email/email-headers.rst:132]: (Agent, Customer, or System)

File: channels/email/email-headers.rst [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   | Example: ``X-Zammad-Article-Sender: System``
+
+      .. warning::
+
+         System emails are indicated like trigger response like entries.
```
Comment (techerella) [inline channels/email/email-headers.rst:137]: would rephrase this sentence, maybe: "System emails are indicated in a similar way as trigger-response-like entries"

File: channels/google/email-headers.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.. note:: 
+
+   EMail header manipulation in Google channels work
```
Comment (techerella) [inline channels/google/email-headers.rst:3]: Email

File: channels/microsoft365/email-headers.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.. note:: 
+
+   EMail header manipulation in Microsoft 365 channels work
```
Comment (techerella) [inline channels/microsoft365/email-headers.rst:3]: Email

File: manage/trigger/how-do-they-work.rst [hunk 5]
```diff
@@ -9,6 +9,11 @@ Triggers are evaluated in alphabetical order, by **name.**
 In some situations triggers might be the wrong choice, see
 :doc:`/manage/trigger/limitations` for more information.
 
+.. hint:: **🤓 E-Mail trigger behavior can be manipulated**
```
Comment (techerella) [inline manage/trigger/how-do-they-work.rst:12]: Email

**Review by techerella — APPROVED** [review]

File: channels/email/email-headers.rst (hunk 2 above)
Comment (MrGeneration) [reply channels/email/email-headers.rst:137]: That's a much better wording, thansk! 🙌
